kasa/tapo/tapodevice.py

```diff
@@ -135,7 +132,7 @@ def mac(self) -> str:
     @property
     def device_id(self) -> str:
         """Return the device id."""
-        return str(self._info.get("device_id"))
+        return str(self._info.get("device_id"))  # type: ignore
```

Comment:
```suggestion
        return str(self._info.get("device_id"))
```
This should not be necessary due to `str` casting?

Reply:
Removed